### PR TITLE
Fix failing Goreleaser due to required CGO_ENABLED flag

### PR DESCRIPTION
**Pull request by adalberht**
**Context**:
- Since we required CGO_ENABLED to be true with the recent WebP support, cross compiling is necessary

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -21,22 +53,22 @@ env:
...
-- replacements:
-    darwin: macOS
-    linux: Linux
-    windows: Windows
-    386: 32-bit
-    amd64: 64-bit
-  format_overrides:
-    - goos: windows
-      format: zip
+  - replacements:
```
Comment (ajatprabha) [inline .goreleaser.yml]: Does this work with this indentation?

File: .goreleaser.yml [hunk 2]
```diff
@@ -21,22 +53,22 @@ env:
...
+  name_template: "checksums.txt"
 snapshot:
   name_template: "{{ .Tag }}-next"
 changelog:
   sort: asc
   filters:
     exclude:
-    - '^docs:'
-    - '^test:'
+      - "^docs:"
```
Comment (ajatprabha) [inline .goreleaser.yml]: How about ignoring `website` too?

File: .goreleaser.yml (hunk 1 above)
Comment (adalberht) [reply .goreleaser.yml]: yes, i didn't format this manually btw, i use prettier plugin in vscode

File: .goreleaser.yml (hunk 2 above)
Comment (adalberht) [reply .goreleaser.yml]: :+1:

File: .goreleaser.yml (hunk 1 above)
Comment (ajatprabha) [reply .goreleaser.yml]: Okay, let's merge this and try to do a release!

**Pull request merged**